Makefile

```
@@ -68,6 +71,8 @@ release: ## Build release binaries
 			GOOS=$${GOOS} GOARCH=$${GOARCH} $(GO) build -ldflags "$(LDFLAGS)" -o $(DIST_DIR)/$${RELEASE_BINARY} ./cmd/$$bin ; \
 		done ; \
 	done
+	$(KUBECTL) create -f config --dry-run=client -o yaml |\
+	  $(SED) 's|ko://github.com/triggermesh/bumblebee/cmd/\(.*\)|gcr.io/triggermesh-private/\1:'${IMAGE_TAG}'|' > $(DIST_DIR)/transformation.yaml
```

```suggestion
	  $(SED) 's|ko://github.com/triggermesh/bumblebee/cmd/\(.*\)|gcr.io/$(IMAGE_REPO)/\1:'${IMAGE_TAG}'|' > $(DIST_DIR)/transformation.yaml
```

Also update `IMAGE_REPO` to be `gcr.io/triggermesh-private`